data/tlg4034/tlg006/tlg4034.tlg006.opp-grc1.xml

```diff
@@ -96,8 +96,13 @@
    <langUsage>
     <language ident="grc">Greek</language>
     <language ident="lat">Latin</language>
+    <language ident="deu">German</language>
    </langUsage>
   </profileDesc>
+    
+    <revisionDesc>
+      <change who="Brittany Hardy" when="2017-06-16">deleted duplicate file opp-grc2, added lang tags, corrected title, updated cts file</change>
+    </revisionDesc>
   
  </teiHeader>
   <text>
```

Comment:
Note: At some point we could think about another structure for this file, addressing sections about one comment, but maybe not now.